exporter/importables_test.go

```diff
@@ -194,7 +194,7 @@ func TestSecretScope(t *testing.T) {
 	d.Set("name", "abc")
 	ic := importContextForTest()
 	name := ic.Importables["databricks_secret_scope"].Name(ic, d)
-	assert.Equal(t, "abc", name)
+	assert.Equal(t, "abc_900150983cd24fb0d6963f7d28e17f72", name)
```

Comment:
not too critical for this PR, but what if we use sha1 and get just first 9 characters from it, like git does - e.g https://github.com/databricks/terraform-provider-databricks/commit/8c16069abaf9843a3955230ab82f1a194199a23f for `8c16069abaf9843a3955230ab82f1a194199a23f`

then the generated conf would look less ugly.